src/pluginManager.ts

```diff
@@ -64,7 +64,7 @@ export interface PluginManagerOptions {
 export class PluginManager {
 
   // name must be prefixed with 'homebridge-' or '@scope/homebridge-'
-  private static readonly PLUGIN_IDENTIFIER_PATTERN = /^((@[\w-]*)\/)?(homebridge-[\w-]*)$/;
+  private static readonly PLUGIN_IDENTIFIER_PATTERN = /^((@[\w-\.]*)\/)?(homebridge-[\w-]*)$/;
```

Comment:
since the lint needs fixing anyway, i wondered if we should disallow a `.` as the first or last character of a scope name.

i think something like this should do it

```
/^(@[a-zA-Z0-9_-]+(\.[a-zA-Z0-9_-]+)*\/)?homebridge-[\w-]+$/
```

Reply:
didn't test mine against the lint! updated:

```
/^(@[\w-]+(\.[\w-]+)*\/)?homebridge-[\w-]+$/
```

Reply:
Yep, that works and makes more sense.  Thanks!

Reply:
Oops, looks like it's missing some brackets.  Will fix up.

Reply:
does the one in the UI repo need fixing?

Reply:
No, it doesn't -- I was in the middle of double checking when you merged it.  It only uses .test() so it's safe even if it's not consistent with homebridge.